projects/core/components/root/root.component.ts

```diff
@@ -28,17 +24,21 @@ import {debounceTime, map} from 'rxjs/operators';
     // So that we do not force OnPush on custom dialogs
     // eslint-disable-next-line @angular-eslint/prefer-on-push-component-change-detection
     changeDetection: ChangeDetectionStrategy.Default,
-    providers: [TUI_IS_MOBILE_RES_PROVIDER],
     encapsulation: ViewEncapsulation.None,
     host: {
         'data-tui-version': TUI_VERSION,
         '[style.--tui-duration.ms]': 'duration',
         '[class._ios]': 'isIOS',
         '[class._android]': 'isAndroid',
+        '[$.class._mobile]': 'isMobileRes$',
```

Comment:
I think AOT will fail due to an unknown property.

Reply:
analogue:
<img width="374" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/2dd034fe-d5c0-44b7-8ff3-dd9cdd9cc37e">

Reply:
I forgot that it doesn't check `host`